drivers/smbus/smbus_utils.h

```diff
@@ -107,4 +110,82 @@ static inline void smbus_init_callback(struct smbus_callback *callback,
  */
 void smbus_loop_alert_devices(const struct device *dev, sys_slist_t *callbacks);
 
+/**
+ * @brief Compute the number of messages required for an SMBus transaction
+ *
+ * If @p flags indicates that the transaction requires packet error checking (PEC),
+ * the number of messages is equal to @p num_msgs, otherwise the number of messages
+ * required is @p num_msgs - 1, since a PEC byte is not required.
+ *
+ * Callers are expected to allocated an array of @ref i2c_msg objects to hold a number of
+ * i2c messages, including one message dedicated to the a PEC byte, whether or not PEC is
```

Comment:
* i2c messages, including one message dedicated to the PEC byte, whether or not PEC is

Reply:
Fixed!